Samples/Widgets/MultipleWidgets/cpp-win32-packaged/SampleWidgetProviderAppPackage/Package.appxmanifest

```diff
@@ -0,0 +1,124 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
```

Comment:
indentation is odd here and others; recommnd: standardize on 4 spaces for indentation

Reply:
should xml and json files have tabs or spaces for identation?

Reply:
Let's use a consistent style; 4 spaces seems standard across our teams (some teams use 2 spaces). Tabs are bad as they might look different in different apps :).